NEWS.md

```diff
@@ -121,6 +121,10 @@ New library features
 Standard library changes
 ------------------------
 
+* `pkg> precompile` is now parallelized, and will run automatically after any `Pkg` action that changes the
+  active manifest. Errors will only throw for direct dependencies listed in the `Project.toml`, and auto-precompillation
+  will remember if a package has errored within the given environment and not retry until it changes. Disable
+  auto-precompillation with `ENV["JULIA_PKG_PRECOMPILE_AUTO"]=0`.
```

Comment:
It might be clearer to split into two items and remove unnecessary details.

```suggestion
* `pkg> precompile` now runs in parallel.
* `pkg> precompile` now gets triggered automatically when the active manifest is changed. The auto-precompilation can be disabled by setting environment variable `ENV["JULIA_PKG_PRECOMPILE_AUTO"]=0`.
```

Reply:
>`pkg> precompile` now runs in parallel.

this makes it sound like it runs along-side something else, which isn't the case

The second wording is  ok, but I want to  include the error handling note

```suggestion
* `pkg> precompile` is now parallelized through depth-first precompillation of dependencies. Errors will only throw for direct dependencies listed in the `Project.toml`.
* `pkg> precompile` is now automatically triggered whenever Pkg changes the active manifest. Auto-precompillation will remember if a package has errored within the given environment and not retry until it changes. Disable auto-precompillation by setting the environment variable `JULIA_PKG_PRECOMPILE_AUTO=0`.
```

Reply:
My argument is that these details are not what normal Julia users would want to concern about. These should be code comments instead of news.

Reply:
Thanks for the feedback. I've tried to improve it a little. 

There is some detail I wanted to take the opportunity to disseminate, but I'm open to reducing it